.github/workflows/ci-testing.yml

```
@@ -34,11 +34,20 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
+          pip install pytest \
+                      pytest-cov \
+                      tensorflow \
+                      libsvm \
+                      pybrisque \
+                      scikit-image
       - name: Test with pytest
         run: |
-          pip install pytest
-          pip install tensorflow
-          pip install libsvm
-          pip install pybrisque
-          pip install scikit-image
-          pytest
+          pytest -x --cov=./ --cov-report=xml
```

I suggest to show the coverage only for `piq` code without tests.
```suggestion
          pytest -x --cov=./piq --cov-report=xml
```

---

Let's try this 😄